### ISSUE TITLE
downsample function and loss function normalization at training stage

### ISSUE THREAD

**Issue by HaoDot** [chosj95/MIMO-UNet#16]
Hi @chosj95 , MIMO-UNet is a fantastic work! THX for sharing!
However, there are some problems confusing me a lot. 
- During the training stage, the different size of blurry images are generated by **nearest interpolation**. However, the corresponding supervision of different size sharp images are generated by **bilinear interpolation**, rather than nearest interpolation again. Intuitively, the interpolation function should be the same. 
- In the paper, function 7&8 have denominators in **loss function for normalization**. However, the code of loss function hasn't the denominators that I have mentioned above. I take it that chances are that the model will have better performance with normalization in loss function.

I am eagerly waitting for your explanation. THX a lot!

**Comment by HaoDot** [comment]
Hi,@chosj95 #6 has solved my problem2. So the remaining problem is just problem1! Waiting for your reply! THX.
